doc/source/ray-overview/examples.rst

```diff
@@ -45,33 +45,40 @@ Ray Examples
             Training 175B Parameter Language Models at 1000 GPU scale with Alpa and Ray
 
     .. grid-item-card:: :bdg-primary:`Blog`
-        :class-item: gallery-item
+        :class-item: gallery-item llm
 
         .. button-link:: https://www.anyscale.com/blog/faster-stable-diffusion-fine-tuning-with-ray-air
 
             Faster stable diffusion fine-tuning with Ray AIR
 
     .. grid-item-card:: :bdg-primary:`Blog`
-        :class-item: gallery-item training serving huggingface
+        :class-item: gallery-item training serving huggingface llm
 
         .. button-link:: https://www.anyscale.com/blog/how-to-fine-tune-and-serve-llms-simply-quickly-and-cost-effectively-using
 
             How to fine tune and serve LLMs simply, quickly and cost effectively using Ray + DeepSpeed + HuggingFace
 
     .. grid-item-card:: :bdg-primary:`Blog`
-        :class-item: gallery-item
+        :class-item: gallery-item llm
 
         .. button-link:: https://www.businessinsider.com/openai-chatgpt-trained-on-anyscale-ray-generative-lifelike-ai-models-2022-12
 
             How OpenAI Uses Ray to Train Tools like ChatGPT
 
     .. grid-item-card:: :bdg-secondary:`Code example`
-        :class-item: gallery-item
+        :class-item: gallery-item llm
 
         .. button-ref:: /ray-air/examples/gptj_deepspeed_fine_tuning
 
             GPT-J-6B Fine-Tuning with Ray AIR and DeepSpeed
 
+    .. grid-item-card:: :bdg-secondary:`Code example`
+        :class-item: gallery-item llm
+
+        .. button-ref:: https://github.com/ray-project/aviary/
```

Comment:
This needs to be `button-link` for it to work I believe :)

Reply:
Sorry. My bad. Just corrected it! 😅